include/ideep/operators/matmul.hpp

```diff
@@ -948,7 +965,7 @@ struct matmul_forward : public dnnl::matmul,
     // Prepare tensor of weight zero point
     static auto wei_zero_point = zero_point_t(1);
     const dim wei_zero_point_size = 1;
-    tensor::desc wei_zero_point_desc = {{wei_zero_point_size}, data_type::s32, {1}};
+    tensor::desc wei_zero_point_desc = {{wei_zero_point_size}, data_type::s32, dims(1, 1)};
```

Comment:
Make it `{wei_zero_point_stride}` instead?

Reply:
OK